### ISSUE TITLE
ffmpeg

### ISSUE THREAD

**Issue by Wallacy** [clearlinux/distribution#433]
Will be good add the ffmpeg (or a Transcoding bundle) as part of the project.

Today's we already have the Gstreamer. But some projects rely on ffmpeg to transcoding. And also, Intel has a good support for ffmpeg libraries.

https://software.intel.com/en-us/articles/accessing-intel-media-server-studio-for-linux-codecs-with-ffmpeg

https://www.intel.com/content/dam/www/public/us/en/documents/white-papers/cloud-computing-quicksync-video-ffmpeg-white-paper.pdf

So, my suggestion is add ffmpeg support with Intel libraries ( libmfx ) as official for Clear Linux project.

**Comment by ahkok** [comment]
Duplicate of #429

**Comment by Wallacy** [comment]
Strange, the older thread is closed because it is duplicated with a newer thread.

Any way, let’s see how this will be done. ;)

**Comment by danboid** [comment]
I personally cannot and will not be using Clear until ffmpeg gets bundled. It is a MUST for the Linux desktop and any semi-serious multimedia. It should be prioritized if Clear is serious about being used as a desktop OS.

**Comment by ahkok** [comment]
Please, let's not get too dramatical about this. The issue isn't that we don't want to offer ffmpeg (we do: we want to offer ffmpeg to all users) but that ffmpeg unfortunately is blocked due to legal issues. 

I encourage you to keep an open mind and help us work on a solution. We're putting time in this effort and hopefully one day it will result in something usable. I am personally attempting to contribute my efforts in the office to accelerate the work that is being done to unblock this, and a positive attitude is far more effective way to get these efforts to completion.

**Comment by flower1024** [comment]
any news on this?
ffmpeg (with x265 and x264 support) is the last thing that blocks me from switching to clear

**Comment by puneetse** [comment]
@flower1024 Updates and discussion on this topic are continuing here: https://github.com/clearlinux/distribution/issues/429
